module/observationSubmissions/helper.js

```diff
@@ -99,7 +181,18 @@ module.exports = class ObservationSubmissionsHelper {
                 if (!observationSubmissionsDocument) {
                     throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND+"or"+messageConstants.apiResponses.SUBMISSION_STATUS_NOT_COMPLETE;
                 }
-
+    
+                let programData = await programsHelper.list({
```

Comment:
@aman-tunerlabs - No need of this code as the flag for the private program is there in the submission document itself.